exporter/trace/cloudtrace.go

```diff
@@ -267,34 +171,44 @@ type Exporter struct {
 
 // InstallNewPipeline instantiates a NewExportPipeline and registers it globally.
 func InstallNewPipeline(opts []Option, topts ...sdktrace.TracerProviderOption) (trace.TracerProvider, func(), error) {
-	tp, flush, err := NewExportPipeline(opts, topts...)
+	tp, shutdown, err := NewExportPipeline(opts, topts...)
 	if err != nil {
 		return nil, nil, err
 	}
 	otel.SetTracerProvider(tp)
-	return tp, flush, err
+	return tp, shutdown, err
 }
 
 // NewExportPipeline sets up a complete export pipeline with the recommended setup
-// for trace provider. Returns provider, flush function, and errors.
+// for trace provider. Returns provider, shutdown function, and errors.
 func NewExportPipeline(opts []Option, topts ...sdktrace.TracerProviderOption) (trace.TracerProvider, func(), error) {
-	exporter, err := NewExporter(opts...)
+	// TODO(suereth): Don't flesh options twice.
+	o := options{Context: context.Background()}
+	for _, opt := range opts {
+		opt(&o)
+	}
+	exporter, err := newExporterWithOptions(&o)
 	if err != nil {
 		return nil, nil, err
 	}
-	tp := sdktrace.NewTracerProvider(append(topts, sdktrace.WithSyncer(exporter))...)
-	return tp, exporter.traceExporter.Flush, nil
+	tp := sdktrace.NewTracerProvider(
+		append(topts,
+			sdktrace.WithBatcher(exporter, o.BatchSpanProcessorOptions...))...)
+	return tp, func() {
```

Comment:
seems like `Shutdown` takes a context on purpose... Should we return a `func(context.Context)` instead of just a `func()`?

Reply:
I think we evaluate that when we evaluate the convenience API.  I want to minimize churn in this change right now.  We have tickets to re-evaluate the convenience API.